app/models/package_manager/elm.rb

```diff
@@ -56,25 +56,16 @@ def self.versions(project, _name)
       end
     end
 
-    def self.dependencies(name, version, project)
-      find_and_map_dependencies(name, version, project)
-    end
-
-    def self.find_dependencies(name, version)
-      url = "https://raw.githubusercontent.com/#{name}/#{version}/elm-package.json"
```

Comment:
This url doesn't work: https://raw.githubusercontent.com/elm/bytes/1.0.2/elm-package.json